src/EditorFeatures/Core/Shared/Utilities/IThreadingContext.cs

```diff
@@ -43,5 +44,22 @@ JoinableTaskFactory JoinableTaskFactory
         {
             get;
         }
+
+        /// <summary>
+        /// Gets a <see cref="CancellationToken"/> indicating that disposal has been requested for the threading
+        /// context.
+        /// </summary>
+        CancellationToken DisposalToken
+        {
+            get;
+        }
```

Comment:
why does thsi file use the 4-line version of interface props?  it's unlike practically teh entire rest of the IDE... very weird.